### PR TITLE
clarify doc

### PR DESCRIPTION
**Pull request by mattreex**
### Improvements
Clarified documentation on study viewer.

### CODE REVIEW

File: docs/study_viewer.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Example configuration:
         "dataType": "dataset",
         "title": "Datasets",           // page title
         "titleField": "name",         // row title
-        "listItemConfig": {  // required
+        "listItemConfig": {
```
Comment (mfshao) [inline docs/study_viewer.md:11]: We probably should fix the code not this doc, `listItemConfig` was intended to be a madatory section 🤦

Comment (mattreex) [reply docs/study_viewer.md:11]: Is it not anymore? This change only makes it clear what is required and what is not. So all fields not labeled as optional are required. It is confusing to have some labelled as required and some labelled as optional and some not labelled as either.

Comment (mfshao) [reply docs/study_viewer.md:11]: I misread your comment, sorry

**Review by mfshao — APPROVED** [review]
lgtm

**Review by paulineribeyre — APPROVED** [review]

**Comment by mfshao** [comment]
what is the relationship of this PR vs https://github.com/uc-cdis/data-portal/pull/817 ?

Ideally we should only keep one of them

**Comment by mattreex** [comment]
> what is the relationship of this PR vs #818 ?
> 
> Ideally we should only keep one of them

I made a new branch from this branch on accident instead from master for #818

**Pull request merged**